# DS 220 Project 2 - Conor Long, Allan Samoilovich
For this project, we will be taking a look at the alcohol consumption from every country. Below are some questions that we will be taking a look at. The data used comes from the FiveThirtyEight GitHub page. The data represents the average alcohol consumption per capita for people age 15 and older. Each unit is per standard drink, so each unit would be one can of beer, one glass of wine, and one shot of spirit. This data is from 2010, so it represents the amount of beer, wine, and spirits consumed on average in that year.

First, we will import the csv file using pandas. We will also import numpy.

In [2]:
import numpy as np
import pandas as pd
drinks = pd.read_csv("drinks.csv")

Q1: Which country has the highest beer servings?

In [6]:
highestBeer = drinks.sort_values(by=['beer_servings'], ascending=False)
highestBeer.head(10)[["country", "beer_servings"]]

country  beer_servings
117         Namibia            376
45   Czech Republic            361
62            Gabon            347
65          Germany            346
98        Lithuania            343
135          Poland            343
188       Venezuela            333
81          Ireland            313
129           Palau            306
140         Romania            297

When running this code, we can see that from all the countrys in the dataset, Namibia has the highest beer servings per person at 376 beers.